scripts/devtron-reference-helm-charts/statefulset-chart_1-0-0/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes (StatefulSet)
+name: statefulset-chart_1-0-0
+version: 1.0.0
```

Comment:
Should we sync the sts chart version with rollout version?
version: 1.0.0 --> 4.18.0